comm/src/main/scala/coop/rchain/comm/transport/GrpcTransportClient.scala

```diff
@@ -162,7 +161,7 @@ class GrpcTransportClient[F[_]: Monixable: Concurrent: Parallel: Log: Metrics](
   ): F[Unit] = {
 
     def timeout(packet: Packet): FiniteDuration =
-      Math.max(packet.content.size().toLong * 5, DefaultSendTimeout.toMicros).micros
+      Math.max(packet.content.size().toLong * 5, networkTimeout.toMicros).micros
```

Comment:
My concern here is that two different algorithms are mixed together. We are choosing from parameter that is relative to packet size and absolute value. This means slower node must opt-in to absolute value.
Although I'm not sure right no what would be easy way to fix it.